testpilot/frontend/static-src/app/views/disable-dialog-view.js

```diff
@@ -48,15 +48,12 @@ export default BaseView.extend({
     e.stopPropagation();
   },
 
-  submit(ev) {
-    ev.preventDefault();
+  submit() {
     this.animateRemove();
     app.sendToGA('event', {
       eventCategory: 'ExperimentDetailsPage Interactions',
       eventAction: 'button click',
-      eventLabel: 'exit survey disabled',
-      newTab: true,
-      outboundURL: ev.target.getAttribute('href')
```

Comment:
Yeah, [the "Take a quick survey" link](https://github.com/mozilla/testpilot/pull/735/files#diff-8c337e6daa565ff7880666fe0c65a403R20) now should natively open that link in a new window/tab since [the `ev.preventDefault()` was removed](https://github.com/mozilla/testpilot/pull/735/files#diff-8c337e6daa565ff7880666fe0c65a403L52). Meanwhile this `sendToGA` call happens passively and doesn't attempt to use `window.open`, which was causing our pop-up blocker issue.